project/SparkBuild.scala

```diff
@@ -271,6 +271,9 @@ object SparkBuild extends PomBuild {
       DefaultMavenRepository,
       Resolver.mavenLocal,
       Resolver.file("ivyLocal", file(Path.userHome.absolutePath + "/.ivy2/local"))(Resolver.ivyStylePatterns)
+    ) ++ Seq(
```

Comment:
Why do we concatenate two `Seq` instead of just adding inside of the `Seq()` call above?